File: Tests/TestData/MultiFileCharacterization/VBToCSResults/ConvertWholeSolution/VbLibrary/AClass.cs
using System.Collections.Generic;
using System.Linq;
using Microsoft.VisualBasic.CompilerServices;

namespace VbLibrary
{
    internal partial class AClass
    {
        public enum NestedEnum
        {
            First
        }

        private Dictionary<int, int> dict = new Dictionary<int, int>();
        private int anInt = 2;
        private int anIntWithNonStaticInitializerReferencingOtherPart;

        public AClass()
        {
            anIntWithNonStaticInitializerReferencingOtherPart = anArrayWithNonStaticInitializerReferencingOtherPart.Length;
            anArrayWithNonStaticInitializerReferencingOtherPart = initialAnArrayWithNonStaticInitializerReferencingOtherPart();
        }

        private void UseOutParameterInClass()
        {
            var x = default(object);
            int argvalue = Conversions.ToInteger(x);
            dict.TryGetValue(1, out argvalue);
            x = argvalue;
        }

        private void UseEnumFromOtherFileInSolution(AnEnum m)
        {
            string nothing = Enumerable.Empty<string>().ToArray()[(int)AnEnum.AnEnumMember];
            switch (m)
            {
                case (AnEnum)(-1):
                    {
                        return;
                    }
                case AnEnum.AnEnumMember:
                    {
                        return;
                    }
            }
        }
    }
}

File: Tests/TestData/MultiFileCharacterization/CSToVBResults/ConvertWholeSolution/VbLibrary/AClass.vb
Friend Class AClass
    Enum NestedEnum
        First
    End Enum

    Private dict As New Dictionary(Of Integer, Integer)
    Private anInt As Integer = 2
    Private anIntWithNonStaticInitializerReferencingOtherPart As Integer = anArrayWithNonStaticInitializerReferencingOtherPart.Length

    Private Sub UseOutParameterInClass()
        Dim x
        dict.TryGetValue(1, x)
    End Sub

    Private Sub UseEnumFromOtherFileInSolution(m As AnEnum)
        Dim [nothing] = Enumerable.Empty(Of String).ToArray()(AnEnum.AnEnumMember)
        Select Case m
            Case -1
                Exit Sub
            Case AnEnum.AnEnumMember
                Exit Sub
        End Select
    End Sub
End Class
